wrappers/android/tools/camera/src/main/java/com/intel/realsense/camera/Streamer.java

```diff
@@ -66,6 +66,7 @@ void configAndStart() throws Exception {
             if(mListener != null)
                 mListener.config(config);
             mPipeline.start(config);
+            try(FrameSet frames = mPipeline.waitForFrames(15000)){} // w/a for l500
```

Comment:
It'd be better to be refactored into the core - we'll need it for all c++/python demos as well

Reply:
I don't think we should do refactoring for a bug, at least not in this PR.